### PR TITLE
adding lints to schema

### PR DESCRIPTION
**Pull request by zakird**

### CODE REVIEW

File: ztag/schema.py [hunk 1]
```diff
@@ -876,6 +876,229 @@ class CensysString(WhitespaceAnalyzedString):
...
+    #   - INFO [failed for info]
+    #   - WARN [failed for warn]
+    #   - FAIL [failed for error]
+    #   - FATAL [test could not complete because cert is broken]
+    #   - UNKNOWN [should never occur]
+    # We don't want to store a string for every lint in elasticsearch because
+    # our index size would explode. Instead we map these to a string:
+    # {
+    #     (reserved, unknown, ne, na, pass) -> null,
+    #     (notice, warning, fail, fatal) -> true
```
Comment (dadrian) [inline ztag/schema.py]: Notice?

File: ztag/schema.py [hunk 2]
```diff
@@ -876,6 +876,229 @@ class CensysString(WhitespaceAnalyzedString):
     "type":Enum(["leaf","intermediate","root"]),
 })
 
+class LintBool(String):
+
+    ES_TYPE = "boolean"
```
Comment (dadrian) [inline ztag/schema.py]: Is this really all we need? Why will this map SUCCESS to `null` in ES?

Comment (zakird) [inline ztag/schema.py]: no. esloader.

File: ztag/schema.py (hunk 1 above)
Comment (zakird) [inline ztag/schema.py]: the new info.

**Pull request merged**